CMakeLists.txt

```
@@ -89,8 +89,12 @@ if ("${isSystemDir}" STREQUAL "-1")
 endif ("${isSystemDir}" STREQUAL "-1")
 set (CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
 
-set(INSTALL_PATH "/opt/prplmesh")
-set(TMP_PATH "/tmp/beerocks")
+if(NOT DEFINED INSTALL_PATH)
+    set(INSTALL_PATH "/opt/prplmesh")
+endif()
```

The normal way of allowing setting from the outside would be to make it a cache variable:
```suggestion
set(INSTALL_PATH "/opt/prplmesh" CACHE PATH "prplMesh installation directory")
```
However, I would much prefer to remove INSTALL_PATH entirely and instead fully rely on CMAKE_INSTALL_PREFIX.

---

> However, I would much prefer to remove INSTALL_PATH entirely and instead fully rely on CMAKE_INSTALL_PREFIX.
There is special task for that #1253
I think that it out of this PR scope